test/smokes/rubocop/expectations.rb

```diff
@@ -17,9 +81,24 @@
         commit: :_, line_hash: "57d3f05dcf6eda436e8767ebacf7058380da0c36", original_line: 23, final_line: 23
       }
     },
+    {
+      message: "Replace unsafe number conversion with number class parsing, instead of using ENV.fetch(\"RAILS_MAX_THREADS\") { 5 }.to_i, use stricter Integer(ENV.fetch(\"RAILS_MAX_THREADS\") { 5 }, 10).",
+      links: %w[
+        https://docs.rubocop.org/rubocop/cops_lint.html#lintnumberconversion
+      ],
+      id: "Lint/NumberConversion",
+      path: "config/puma.rb",
+      location: { start_line: 7, start_column: 17, end_line: 7, end_column: 57 },
+      object: { severity: "warning", corrected: false },
+      git_blame_info: {
+        commit: :_, line_hash: "87a9e59b8f03dcdb86ec1b5f1e293d7f72e9a37b", original_line: 7, final_line: 7
+      }
+    },
     {
       message: "Shadowing outer local variable - `v`.",
-      links: %w[https://docs.rubocop.org/rubocop/cops_lint.html#lintshadowingouterlocalvariable],
+      links: %w[
+        https://docs.rubocop.org/rubocop/cops_lint.html#lintshadowingouterlocalvariable
+      ],
```

Comment:
[suggestion] Can you minimize the diff?

```suggestion
      links: %w[https://docs.rubocop.org/rubocop/cops_lint.html#lintshadowingouterlocalvariable],
```

Reply:
OK, upadted links.